client/components/mobile/Footer.jsx

```diff
@@ -1,20 +1,42 @@
 import React from 'react';
 import styled from 'styled-components';
+import PropTypes from 'prop-types';
 import { prop, remSize } from '../../theme';
 
+
 const background = prop('MobilePanel.default.background');
 const textColor = prop('primaryTextColor');
 
-const Footer = styled.div`
+const FooterWrapper = styled.div`
   position: fixed;
   width: 100%;
+  bottom: 0;
+`;
+
+const FooterContent = styled.div`
   background: ${background};
   color: ${textColor};
   padding: ${remSize(12)};
   padding-left: ${remSize(32)};
-  z-index: 1;
-
-  bottom: 0;
 `;
 
+const Footer = ({ before, children }) => (
+  <FooterWrapper>
+    {before}
```

Comment:
I think it's strange to have some of the footer content not wrapped in an extra container and some of it in a container. Like, what do we really want the Footer to represent? Content that's at the bottom of the screen. I think all of the children of `<Footer>` should be full-width and layer on top of each other, and any of those Components can have children if necessary. Therefore, I suggest getting rid of `before` and just having `children`.

Reply:
Good point. This should be an easy fix